### DISCUSSION TITLE
Downloaded, how to put into dreamweaver

### DISCUSSION THREAD

**Discussion by lanchadesigns** [thecarnie/cycle2#7] [Q&A] [answered]
I am new to dreamweaver and am trying to create a slideshow.  I downloaded your cycle2 but am not sure how to move this information into my site.  Is there one file I can move that will install everything I need into my code?

**Comment by petertwise** [comment, marked as answer]
Hi @lanchadesigns  -

This repository is a fork of the original Cycle2, which is no longer supported by its original author. You should start be reviewing the original documentation here:
https://jquery.malsup.com/cycle2/ 
https://jquery.malsup.com/cycle2/demo/basic.php

For the bare basics, you just need to make sure you have jQuery installed already, and then add the `jquery.cycle2.min.js` file from the `build` directory to your project.
